libs/datamodel/connectors/sql-datamodel-connector/src/mysql_datamodel_connector.rs

```diff
@@ -257,9 +261,9 @@ impl Connector for MySqlDatamodelConnector {
             MEDIUM_TEXT_TYPE_NAME => MySqlType::MediumText,
             LONG_TEXT_TYPE_NAME => MySqlType::LongText,
             DATE_TYPE_NAME => MySqlType::Date,
-            TIME_TYPE_NAME => MySqlType::Time(args.first().cloned()),
-            DATETIME_TYPE_NAME => MySqlType::DateTime(args.first().cloned()),
-            TIMESTAMP_TYPE_NAME => MySqlType::Timestamp(args.first().cloned()),
+            TIME_TYPE_NAME => MySqlType::Time(parsed_args.first().cloned()),
```

Comment:
We could also do validation in here, maybe? 

```
            LONG_TEXT_TYPE_NAME => match parsed_args.as_slice() {
                [] => MySqlType::LongText,
                _ => todo!("Maybe return an error?"),
            },
            TIME_TYPE_NAME => match parsed_args.as_slice() {
                [fractions] => MySqlType::Time(Some(fractions)),
                _ => todo!("Return an error"),
            }
```

Reply:
I mean, this is already a bigger refactor, but we're matching anyhow here, so would make sense if the param count validation would also be here?

Reply:
yes, for consistency it would also be good here 👍